src/css/layout/fixedDataTableCellGroupLayout.css

```diff
@@ -27,4 +27,5 @@
 .fixedDataTableCellGroupLayout/cellGroupWrapper {
   position: absolute;
   top: 0;
+  touch-action: none;
```

Comment:
Since this could have unexpected consequences, I'm going to wait til v1.0 to pull it in.

If you want to rebase your changes off that branch, you should be able to use the beta release with it.  Otherwise, I'm open to taking the other changes, and you can add this css to your style sheet until v1 is released.

Reply:
Sounds good. Removing the _touch-action: none;_ rules from this PR. I will add that CSS to my stylesheet and wait for v1.